### PR TITLE
Added showing error message on invalid url entry

### PR DESCRIPTION
**Pull request by mustafapc19**
Ref: https://github.com/nocodb/nocodb/issues/2980#issuecomment-1208348855

Screenshot:
image
image

### CODE REVIEW

**Review by wingkwong — CHANGES_REQUESTED** [review]
1. Validation should be only performed when users have selected `Accept only valid URL`. Now validation is triggered anyway.

image
 
2. The value shouldn't be saved if the URL is invalid. Now error message is shown but the value is stored. (you can refresh / reload to see it)

**Comment by mustafapc19** [comment]
@wingkwong Regarding 2nd point can you tell me the steps to reproduce? In my case when I am reloading/refreshing its reverting to the last saved/validated value.

**Comment by wingkwong** [comment]
Now it seems fine for the second point. However, with `Accept only valid URL` enabled, I couldn't set it empty.

**Comment by wingkwong** [comment]
Another thought 

- Create a URL column with `Accept only valid URL` disabled
- Input data with some invalid URL
- Edit the column. Enabled `Accept only valid URL`

In this case, I think it'd be great if those cells have some indicators to show that they are invalid so that users can easily spot them to change the values.

**Comment by mustafapc19** [comment]
@wingkwong Have added ui indication for URL being invalid. Attached the screenshot in the PR description.

File: packages/nc-gui-v2/components/cell/Url.vue [hunk 1]
```diff
@@ -35,14 +41,37 @@ const url = computed(() => {
...
 
-  <nuxt-link v-else-if="isValid" class="text-sm underline hover:opacity-75" :to="url" target="_blank">{{ value }} </nuxt-link>
+    <span v-else class="w-9/10 overflow-ellipsis overflow-hidden">{{ value }}</span>
 
-  <span v-else>{{ value }}</span>
+    <div v-if="!isValid && value?.length && !editEnabled" class="mr-1 w-1/10">
+      <a-tooltip placement="top">
+        <template #title> Invalid URL </template>
+        <div class="flex flex-row items-center">
+          <MaterialSymbolsWarning />
```
Comment (wingkwong) [inline packages/nc-gui-v2/components/cell/Url.vue:70]: Suggest to color it.

image

**Review by wingkwong — COMMENTED** [review]
`qweq` shouldn't be a hyperlink. (Accept only valid URL: false)

v2:

image

v1: 
image

**Review by wingkwong — APPROVED** [review]

**Pull request merged**